content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,6 +38,45 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
+## Guidelines for contributing content
+
+Kubernetes documentation only hosts content from projects within the
+ `kubernetes` or `kubernetes-sigs` GitHub organizations or from current CNCF
+  projects ([Graduated/Incubating](https://www.cncf.io/projects/), [Sandbox](https://www.cncf.io/sandbox-projects/)).
```

Comment:
I would actually argue *against* the Kubernetes documentation hosting *any* content other than content for projects in the kubernetes or kubernetes-sigs GitHub organizations. Kubernetes documentation should be for Kubernetes things only.

Making this decision would allow a clear line to be drawn and also allow the docs team to avoid the sticky issue around whether to include CNCF sandboxed projects (which have a much lower bar) in documentation.

Reply:
I tried to make it clear that project docs belong in the project's docs, with Kubernetes dos linking to the project's docs. I may have overdone "...a CNCF project or a project in the kubernetes or kubernetes-sigs GitHub organizations..."  - I can be somewhat pedantic.